Model/Method/Knaken.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please send an email
+ * to support@buckaroo.nl so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+
+namespace Buckaroo\Magento2\Model\Method;
+
+class Knaken extends AbstractMethod
+{
+    /** Payment Code*/
+    const PAYMENT_METHOD_CODE = 'buckaroo_magento2_knaken';
+
+    /** @var string */
+    public $buckarooPaymentMethodCode = 'knaken';
+
+    /** @var string */
+    protected $_code = self::PAYMENT_METHOD_CODE;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getOrderTransactionBuilder($payment)
+    {
+        $transactionBuilder = $this->transactionBuilderFactory->get('order');
+
+        $services = [
+            'Name'    => 'Knaken',
+            'Action'  => $this->getPayRemainder($payment, $transactionBuilder),
```

Comment:
does this method allow pay remainders ?

Reply:
@harli91 & @LucianTuriacArnia,

I just asked the Buckaroo Payment Services team and they confirmed that PayRemainder is not yet supported for Knaken Settle (Collecting). It was already added for Knaken Processing, but not yet for Collecting.
However they informed me that they will fix that today, just matter of some queries.